pyproject.toml

```diff
@@ -429,7 +429,96 @@ norecursedirs = [
+    "ignore:'telnetlib' is deprecated and slated for removal in Python 3.13:DeprecationWarning:ndms2_client.connection",
+    # https://github.com/grahamwetzler/smart-meter-texas/pull/143
+    "ignore:ssl.OP_NO_SSL\\*/ssl.OP_NO_TLS\\* options are deprecated:DeprecationWarning:smart_meter_texas",
+    # https://github.com/pytest-dev/pytest-cov/issues/557 - v4.1.0
+    # Should resolve itself once pytest-xdist 4.0 is released and the option is removed
+    "ignore:The --rsyncdir command line argument and rsyncdirs config variable are deprecated:DeprecationWarning:xdist.plugin",
+    # https://github.com/eclipse/paho.mqtt.python/blob/v1.6.1/src/paho/mqtt/client.py#L788-L791
+
+    # -- fixed, waiting for release / update
+    # https://github.com/gurumitts/pylutron-caseta/pull/143 - >0.18.1
+    "ignore:ssl.PROTOCOL_TLSv1_2 is deprecated:DeprecationWarning:pylutron_caseta.smartbridge",
+    # https://github.com/Danielhiversen/pyMillLocal/pull/8 - >=0.3.0
+    "ignore:with timeout\\(\\) is deprecated:DeprecationWarning:mill_local",
+    # https://github.com/home-assistant/core/pull/98619 - update botocore to >=1.31.17
+    "ignore:'cgi' is deprecated and slated for removal in Python 3.13:DeprecationWarning:botocore.utils",
+    "ignore:'urllib3.contrib.pyopenssl' module is deprecated and will be removed in a future release of urllib3 2.x:DeprecationWarning:botocore.httpsession",
+
+    # -- not helpful
+    # pyatmo.__init__ imports deprecated moduls from itself - v7.5.0
+    "ignore:The module pyatmo.* is deprecated:DeprecationWarning:pyatmo",
+
+    # -- unmaintained projects, last release about 2+ years
+    # https://pypi.org/project/agent-py/ - v0.0.23 - 2020-06-04
+    "ignore:with timeout\\(\\) is deprecated:DeprecationWarning:agent.a",
+    # https://pypi.org/project/aiomodernforms/ - v0.1.8 - 2021-06-27
+    "ignore:with timeout\\(\\) is deprecated:DeprecationWarning:aiomodernforms.modernforms",
+    # https://pypi.org/project/directv/ - v0.4.0 - 2020-09-12
+    "ignore:with timeout\\(\\) is deprecated:DeprecationWarning:directv.directv",
+    # https://pypi.org/project/emulated-roku/ - v0.2.1 - 2020-01-23 (archived)
+    "ignore:loop argument is deprecated:DeprecationWarning:emulated_roku",
+    # https://pypi.org/project/foobot_async/ - v1.0.0 - 2020-11-24
+    "ignore:with timeout\\(\\) is deprecated:DeprecationWarning:foobot_async",
+    # https://pypi.org/project/lark-parser/ - v0.12.0 - 2021-08-30 -> moved to `lark`
+    # https://pypi.org/project/commentjson/ - v0.9.0 - 2020-10-05
+    # https://github.com/vaidik/commentjson/issues/51
+    # Fixed upstream, commentjson depends on old version and seems to be unmaintained
+    "ignore:module '(sre_parse|sre_constants)' is deprecate:DeprecationWarning:lark.utils",
+    # https://pypi.org/project/lomond/ - v0.3.3 - 2018-09-21
+    "ignore:ssl.PROTOCOL_TLS is deprecated:DeprecationWarning:lomond.session",
+    # https://pypi.org/project/paho-mqtt/ - v1.6.1 - 2021-10-21
```

Comment:
I hope paho-mqtt isn't unmaintained. It's a corner stone for many things MQTT.

Reply:
I hope so too but at the moment it isn't looking good. The last commit was 7 months ago. I can remove the entry here if you like.
https://github.com/eclipse/paho.mqtt.python

Reply:
It doesn't look good. Found this post on the mailing list: https://www.eclipse.org/lists/paho-dev/msg04742.html
Anyway, the deprecation is also tracked upstream, so moved the entry to the another section.
- https://github.com/eclipse/paho.mqtt.python/issues/653
- https://github.com/eclipse/paho.mqtt.python/pull/665